riotfile.py

```diff
@@ -180,6 +180,80 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
             command="pytest {cmdargs} tests/contrib/logging",
             pys=select_pys(),
         ),
+        Venv(
+            name="falcon",
+            command="pytest {cmdargs} --ignore tests/contrib/falcon/test_autopatch.py tests/contrib/falcon",
+            venvs=[
+                # Falcon 1.x
+                # Python 2.7+
+                Venv(
+                    pys=select_pys(),
+                    pkgs={
+                        "falcon": [
+                            "~=1.4.1",
+                            "~=1.4",  # latest 1.x
+                        ]
+                    },
+                ),
+                Venv(
+                    command="python tests/ddtrace_run.py pytest {cmdargs} tests/contrib/falcon/test_autopatch.py",
+                    pys=select_pys(),
+                    pkgs={
+                        "falcon": [
+                            "~=1.4.1",
+                            "~=1.4",  # latest 1.x
+                        ]
+                    },
+                    env={"DD_SERVICE": "my-falcon"},
+                ),
+                # Falcon 2.x
+                # Python 3.5+
+                Venv(
+                    pys=select_pys(min_version="3.5"),
+                    pkgs={
+                        "falcon": [
+                            "~=2.0.0",
+                            "~=2.0",  # latest 2.x
+                        ]
+                    },
+                ),
+                Venv(
+                    command="python tests/ddtrace_run.py pytest {cmdargs} tests/contrib/falcon/test_autopatch.py",
+                    pys=select_pys(min_version="3.5"),
+                    pkgs={
+                        "falcon": [
+                            "~=2.0.0",
+                            "~=2.0",  # latest 2.x
+                        ]
+                    },
+                    env={"DD_SERVICE": "my-falcon"},
+                ),
+                # Falcon 3.x
+                # Python 3.5+
+                Venv(
+                    pys=select_pys(min_version="3.5"),
+                    pkgs={
+                        "falcon": [
+                            "~=3.0.0",
+                            "~=3.0",  # latest 3.x
+                            latest,
+                        ]
+                    },
+                ),
```

Comment:
Same here, not sure how exactly riot/tox works so just curious if it's necessary since this doesn't actually run any commands

Reply:
it'll inherit the base command from the parent venv. I can move the command into these to make it more clear if that'll help?